library/stringUtils.sol

```diff
@@ -24,4 +24,32 @@ library StringUtils {
     function equal(string _a, string _b) returns (bool) {
         return compare(_a, _b) == 0;
     }
+    /// @dev Finds the first occurrence of _b in _a and returns its index or -1 if failure
+    function indexOf(string _a, string _b) returns (int) // _a = string to search, _b = string we want to find
+    {
+    	bytes memory a = bytes(_a);
+    	bytes memory b = bytes(_b);
+    	if(a.length < 1 || b.length < 1 || (b.length > a.length)) 
```

Comment:
Actually I just tried if (haystack == needle) return 0, but the compiler says strings and bytes memory can't use the == operator.